### PR TITLE
Fix small typos

### PR DESCRIPTION
**Pull request by dalizard**
A pair of small fixes for the EUnit documentation.

### CODE REVIEW

File: lib/eunit/doc/overview.edoc [hunk 1]
```diff
@@ -994,7 +994,7 @@ specified node. `local' means that the current process will handle both
 setup/teardown and running the tests - the drawback is that if a test
 times out so that the process is killed, the <em>cleanup will not be
 performed</em>; hence, avoid this for persistent fixtures such as file
-operations. In general, 'local' should only be used when:
+operations. In general, `local` should only be used when:
```
Comment (bjorng) [inline lib/eunit/doc/overview.edoc]: This is wrong. The second single quote should **not** be changed. Should be:

    `local'

Please amend your commit.

Comment (dalizard) [inline lib/eunit/doc/overview.edoc]: @bjorng Indeed. I've updated the wrong notation to the proper one - `local'.

**Comment by bjorng** [comment]
Thanks!

But please squash the commits into a single commit.

**Comment by dalizard** [comment]
@bjorng Commits squashed 👍

**Pull request merged**